app/sql.js

```diff
@@ -837,11 +852,27 @@ async function updateToLokiSchemaVersion1(currentVersion, instance) {
   const publicChatData = {
     ...baseData,
     id: 'publicChat:1@chat.lokinet.org',
-    server: 'https://chat.lokinet.org',
+    server: 'chat.lokinet.org',
     name: 'Loki Public Chat',
     channelId: '1',
   };
 
+  const { server, token } = lokiPublicServerData;
+
+  await instance.run(
+    `INSERT INTO server_tokens (
```

Comment:
Can we reduce duplication by reusing `savePublicServerToken(lokiPublicServerData)`?

Reply:
We actually *can't* use that (would have liked to use `saveConversation` instead of the `initConversation` function above as well) but those functions use the `db` object which isn't instantiated when the schema updates are running, so we need to use `instance` manually